waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -276,4 +286,12 @@ proc setPeer*(ws: WakuSwap, peer: PeerInfo) =
   ws.peerManager.addPeer(peer, WakuSwapCodec)
   waku_swap_peers_count.inc()
 
+proc getPeerConnectionStatus*(ws: WakuSwap, peerId : PeerId) : PeerConnectionStatus = 
+  let balance = ws.accounting[peerId];
```

Comment:
What if peerId doesn't exist?

Reply:
Thanks. This was actually the main cause of the problems I had earlier. Later figured it out with trace logs enabled.